bundle: set RELATED_IMAGE_PODVM_OCI

snir911 opened the pull request:
This required also some configuration changes in order to be able to pull the image in-job

snir911 committed:
bundle: set RELATED_IMAGE_PODVM_OCI

Signed-off-by: Snir Sheriber <ssheribe@redhat.com>

bpradipt commented on config/peerpods/podvm/lib.sh:
```diff
@@ -719,6 +719,21 @@ retry_command() {
+# It is usually used for test unreleased prebuilt images
+# The configuration is appended to the global /etc/containers/registries.conf file
+function set_containers_resgistries_conf() {
+    if [[ -z "${CONTAINER_REGISTRIES_CONF}" ]]; then
```
How about using imagedigestmirrorsets and imagetagmirrorsets if it exists ?

bpradipt replied:
or using hostPath mount for the node's `/etc/containers/registries.conf` ?

snir911 committed:
podvm: mount host's containers configs

in order to allow in-job podman & skopeo to mimic host's
containers configuration

including:
- registries config (needed for mirroring)
- auths config (needed for registry credentials)
- policy config (needed for signing policy)

ignoring regauth as it should be deprecated in the future

Signed-off-by: Snir Sheriber <ssheribe@redhat.com>

bpradipt reviewed (approved):
/lgtm
